actions/utils/parse_workspace/action.yml

```diff
@@ -21,28 +21,4 @@ runs:
 
     - id: load_workspace_file
       shell: bash
-      run: |
-        python -c "
-        import yaml, json
-        import os
-
-        workspace = '${{ inputs.dagster_cloud_file }}'
-        secrets_set = bool(os.getenv('DAGSTER_CLOUD_API_TOKEN'))
-
-        with open(workspace) as f:
-            workspace_contents = f.read()
-        workspace_contents_yaml = yaml.safe_load(workspace_contents)
-
-        output_obj = [
-            {
-                'name': location['location_name'],
-                'directory': location.get('build', {'directory': '.'}).get('directory'),
-                'build_folder': location.get('build', {'directory': '.'}).get('directory'),
-                'registry': location.get('build', {'directory': '.'}).get('registry'),
-                'location_file': str(workspace),
-            }
-            for location in workspace_contents_yaml['locations']
-        ]
-        print(f'::set-output name=build_info::{json.dumps(output_obj)}')
-        print(f'::set-output name=secrets_set::{json.dumps(secrets_set)}')
-        "
+      run: "python /parse_workspace.py ${{ inputs.dagster_cloud_file }}"
```

Comment:
Note this runs in github's docker image, not ours, so `/parse_workspace.py` is not available. I assume this is why it was written inline.

However you might be able to find it [using](https://docs.github.com/en/actions/learn-github-actions/environment-variables) `$GITHUB_ACTION_PATH`.
Eg something like `$GITHUB_ACTION_PATH/..../src/parse_workspace.py`

Reply:
Ah, good note. I just assumed it was because I hadn't rebuilt/pushed the image yet.